portal-bridge/src/census.rs

```diff
@@ -256,7 +278,7 @@ impl Network {
                     None
                 }
             })
-            .take(ENRS_RESPONSE_LIMIT)
+            .take(self.enr_offer_limit)
```

Comment:
So I guess whatever order `peers` is in is the order that we tend to prefer offering content to. Maybe we should shuffle (or partial_shuffle) before taking here, in order to push to a broader mix of peers

Reply:
Well, the order in this case should always be the same. Sorted by closest to content id. Each census has the same view of the network (eg 100% of peers), so each census will be gossiping the same content to the same peers. Which is not great. But. Eventually, the state bridges (as we "scale" horizontally) will only gossip content that is close their own node id. In which case, we lose this redundancy but maintain the feature (which is desired) of gossiping content to their closest nodes on the network (which would be lost if we shuffle). I guess, `partial_shuffle` is an option... and would mitigate the grey area that we're currently in (aka until we're horizontally scaled). But I'm not sure it's that important overall... though I could be convinced otherwise